### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [LibRedirect/extension](https://hosted.weblate.org/projects/libredirect/extension/).

Current translation status:

Weblate translation status

### CODE REVIEW

File: src/_locales/es/messages.json [hunk 1]
```diff
@@ -2,5 +2,214 @@
...
+    "tor": {
+        "message": "Tor",
+        "description": "used in the settings page"
+    },
+    "theme": {
+        "message": "Tema",
+        "description": "used in the settings page"
+    },
+    "light": {
+        "message": "Claro",
```
Comment (EdwardLangdon) [inline src/_locales/es/messages.json:47]: I need this to be checked by the Author.

File: src/_locales/es/messages.json [hunk 2]
```diff
@@ -2,5 +2,214 @@
...
+    "onlyNotEmbedded": {
+        "message": "solo los no incrustados",
+        "description": "used in the settings page"
+    },
+    "instanceIsOff": {
+        "message": "La instancia está offline",
+        "description": "used in instance_offline.html"
+    },
+    "searchNote": {
+        "message": "Nota: Utilice la búsqueda en todo su potencial haciendo de LibRedirect su motor de búsqueda predeterminado.",
```
Comment (EdwardLangdon) [inline src/_locales/es/messages.json:196]: This too.

**Review by EdwardLangdon — CHANGES_REQUESTED** [review]

File: src/_locales/es/messages.json (hunk 1 above)
Comment (EdwardLangdon) [inline src/_locales/es/messages.json:47]: I am unsure of this alone. Everything seems fines.

Comment (EdwardLangdon) [reply src/_locales/es/messages.json:47]: The Light string alone.

Comment (ManeraKai) [reply src/_locales/es/messages.json:47]: I think it's correct: https://st.privacydev.net/?engine=google&text=Tema%20Claro&sl=es&tl=en

Comment (EdwardLangdon) [reply src/_locales/es/messages.json:47]: Okay, Great.

**Pull request merged**